### ISSUE TITLE
Robin Go Live for ISO

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#7348]
Go Live mid Nov

**Comment by moorerst** [comment]
Wendy is OOO this week will follow up with her next week to determine if anything else is needed from us.

**Comment by moorerst** [comment]
Update from ISO team: Thanks so much for reaching out, Stephanie. Karl was going to reach out with a few questions, then we should be ready to go. Good news – we were able to get an Active Directory entry for our conference room, so it can now be a schedulable resource (that will be one of his questions). 😉 He’s going to be out for the next couple of days, but he / I will reach out then.

**Comment by moorerst** [comment]
check in with ISO team:

Response from Karl:
Thank you for checking in. I think all is in order right now. We are using Robin. Even though our office is still doing a fair amount of TLC, the platform is very handy, especially the mobile app. 

Thanks again to Diana for assisting with the final steps we needed to be fully up and running.  I am sure there will be a question on use from time to time, but we are good to go.

ISO team is up and running with Robin. Closing this issue.
